### PR TITLE
feat(oui-checkbox): description attribute

### PR DESCRIPTION
**Pull request by jimmyfortinx**
Signed-off-by: Jimmy Fortin <jimmy.fortin@corp.ovh.com>

http://angular_feat_checkbox_description.ui-kit.ovh/#!/oui-angular/checkbox

### CODE REVIEW

File: packages/oui-checkbox/src/checkbox.html [hunk 1]
```diff
@@ -8,6 +8,7 @@
          ng-disabled="$ctrl.disabled">
   <label for="{{:: $ctrl.id }}" class="oui-checkbox__label-container">
     <span class="oui-checkbox__label" ng-bind="$ctrl.text"></span>
+    <span class="oui-checkbox__description" ng-bind="$ctrl.description" ng-if="$ctrl.description"></span>
```
Comment (jleveugle) [inline packages/oui-checkbox/src/checkbox.html]: The label must be explicit and concise, 'cause the label is read when the linked input is focus.

So you will hear : "Label Description BLABLABLABLABLABLABLA - Text field - Required"

You should consider aria-describedby and keep a simple label :)

Comment (AxelPeter) [reply packages/oui-checkbox/src/checkbox.html]: Maybe with a structure like:
```
<div>
  <input>
  <label></label>
  <span>></label>
</div>
```

Comment (AxelPeter) [reply packages/oui-checkbox/src/checkbox.html]: Forget my upper comment,
https://developer.mozilla.org/fr/docs/Accessibilit%C3%A9/ARIA/Techniques_ARIA/Utiliser_l_attribut_aria-describedby

Like said @jleveugle `aria-labelledby` and `aria-describedby` seems to resolve this issue.

Comment (jimmyfortinx) [reply packages/oui-checkbox/src/checkbox.html]: Fixed, I used `aria-labelledby` and `aria-describedby` since we need the description to be clickable.

**Pull request merged**